zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java

```diff
@@ -416,10 +413,12 @@ public String getFormType(String noteId, String className) throws TException {
   }
 
   @Override
-  public List<String> completion(String noteId, String className, String buf, int cursor)
+  public List<InterpreterCompletion> completion(String noteId, String className, String buf,
+                                                int cursor)
```

Comment:
Yeah I did as you said and repushed. If I did wrong, please let me know :) 
The reason why I did like this is this line is over 100 line length. So I cut this line on purpose.